src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,204 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.CustomRegistrationException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "valid_login";
+    private static final String MINIMUM_VALID_LOGIN_SIZE = "little";
+    private static final String LOGIN_FIVE_LENGTH = "login";
+    private static final String NOT_ENOUGH_LOGIN_CHARACTERS = "log";
+    private static final String LOGIN_EMPTY = "";
+    private static final String VALID_PASSWORD = "valid_password";
+    private static final String MINIMUM_VALID_PASSWORD_SIZE = "123456";
+    private static final String PASSWORD_FIVE_LENGTH = "12345";
+    private static final String PASSWORD_TOO_SMALL = "123";
+    private static final String PASSWORD_EMPTY = "";
+    private static final int VALID_AGE = 21;
+    private static final int MINIMUM_AGE_ALLOWED = 18;
+    private static final int AGE_NOT_ALLOWED = 17;
+    private static final int AGE_MINUS_VALUE = -12;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @Test
+    void register_userEverythingIsValid_Ok() {

```

Comment:
First letter after _ should be lower case. Make sure that you fix it everywhere.
```suggestion
    void register_userEverythingIsValid_ok() {

```